jkube-kit/jkube-kit-quarkus/src/test/java/org/eclipse/jkube/quarkus/enricher/QuarkusHealthCheckEnricherTest.java

```diff
@@ -162,56 +167,39 @@ public void create_withQuarkus2AndAbsoluteNonApplicationRootPathProperty_shouldR
     assertLivenessReadinessStartupProbes(klb, tuple("HTTP", "/absolute/health/liveness", "HTTP", "/absolute/health/ready", "HTTP", "/absolute/health/started"));
   }
 
-  @Test
-  public void create_withQuarkus3AndAbsoluteLivenessPathProperty_shouldReturnAbsoluteCustomizedPaths() {
-    // Given
-    withSmallryeDependency();
-    withQuarkus("3.1337.3.Final");
-    properties.put("quarkus.smallrye-health.liveness-path", "/absolute/liveness");
-    // When
-    new QuarkusHealthCheckEnricher(context).create(PlatformMode.kubernetes, klb);
-    // Then
-    assertLivenessReadinessStartupProbes(klb, tuple("HTTP", "/absolute/liveness", "HTTP", "/q/health/ready", "HTTP", "/q/health/started"));
-  }
-
-  @Test
-  public void create_withQuarkus3AndAbsoluteReadinessPathProperty_shouldReturnAbsoluteCustomizedPaths() {
-    // Given
-    withSmallryeDependency();
-    withQuarkus("3.1337.3.Final");
-    properties.put("quarkus.smallrye-health.readiness-path", "/absolute/readiness");
-    // When
-    new QuarkusHealthCheckEnricher(context).create(PlatformMode.kubernetes, klb);
-    // Then
-    assertLivenessReadinessStartupProbes(klb, tuple("HTTP", "/q/health/live", "HTTP", "/absolute/readiness", "HTTP", "/q/health/started"));
-  }
-
-  @Test
-  public void create_withQuarkus1_0AndAbsoluteLivenessPathProperty_shouldIgnoreLeadingSlash() {
+  @DisplayName("Create probes with different Quarkus version and properties tests")
+  @ParameterizedTest(name = "{0}")
```

Comment:
Why did you choose to combine some of the test cases into a parameterized test and not the others?

It might be a good exercise to write the `@ParameterizedTest(name = "{0}")` using placeholders instead of a full description. Then it might be easier to identify those tests that can be combined and those that shouldn't be.

For this complete test suite, I guess that test cases could be combined in pairs with still having some meaning.